device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClientConfig.java

```diff
@@ -627,32 +590,6 @@ void setOperationTimeout(long timeout) throws IllegalArgumentException
         this.operationTimeout = timeout;
     }
 
-    /**
-     * Getter for the device operation timeout
-     * @return the amount of time, in milliseconds, before any device operation expires
-     */
-    public long getOperationTimeout()
-    {
-        //Codes_SRS_DEVICECLIENTCONFIG_34_032: [This function shall return the saved operation timeout.]
-        return this.operationTimeout;
-    }
-
-    public ProductInfo getProductInfo()
-    {
-        //Codes_SRS_DEVICECLIENTCONFIG_34_040: [This function shall return the saved product info.]
-        return this.productInfo;
-    }
-
-    public void setProxy(ProxySettings proxySettings)
```

Comment:
Using lombok setter to replace this renames it to "setProxySettings" which is more appropriate since the getter also refers to it as "ProxySettings" rather than just "Proxy"